src/main/java/bowling/domain/state/FrameStateBonus.java

```diff
@@ -26,7 +28,7 @@ public FrameStateBonus(List<Pinfall> pinfalls) {
     @Override
     public FrameState roll(Pinfall pinfall) {
         if (!isRollable()) {
-            throw new IllegalArgumentException("공을 던질 수 없습니다");
+            throw new IllegalRollingSequenceException("공을 던질 수 없습니다");
```

Comment:
```suggestion
            throw new IllegalRollingSequenceException();
```

Custom Exception 안에 메시지를 정의하면, 다른 예외에서는 "공을 굴릴 수 없습니다" 메시지를 던질 때
이 아이만 혼자서 "던질 수 없습니다" 라는 상황을 방지하고, 통일성을 줄 수도 있습니다.